### DISCUSSION TITLE
Still can't figure out how to add myself as an admin

### DISCUSSION THREAD

**Discussion by Tymron** [EvanMulawski/BeamMP.EnvSync#24] [Q&A] [answered]
I've read the instructions 10 times and checked all the comments but it still doesnt work. Any ideas? 
![2](https://github.com/EvanMulawski/BeamMP-EnvSync/assets/142936439/a78e2684-04aa-47d6-b48b-8e8ec3125534)
![3](https://github.com/EvanMulawski/BeamMP-EnvSync/assets/142936439/c6ad983f-34db-4cd3-b184-8fea76bec89d)
![1](https://github.com/EvanMulawski/BeamMP-EnvSync/assets/142936439/a6b13149-d88b-487a-a866-84584083f136)

**Comment by EvanMulawski** [comment, marked as answer]
@Tymron You added it inside a comment block (`--[[` starts a multi-line comment).

Add that line under the example line `--options.admins.beammp_123456...` instead.

**Comment by Tymron** [comment]
Oh I get it now, thank you!
